l10n_br_account_payment_brcobranca/parser/cnab_file_parser.py

```diff
@@ -620,6 +622,14 @@ def get_move_vals(self):
         of statement.
         :return: dict of vals that represent additional infos for the statement
         """
+        for row in self.cnab_return_events:
+            if "real_payment_date" in row and row["real_payment_date"]:
+                self.move_date = row["real_payment_date"]
+                break
+            elif "occurrence_date" in row:
+                self.move_date = row["occurrence_date"] + datetime.timedelta(
+                    days=self.journal.floating_days
```

Comment:
Todas as Ocorrências que não são de Liquidação, campo real_payment_date, devem somar a Data referente ao 'Dias para Compensação de Credito/Debito', tem certeza? Quando for um Ocorrência de Protesto ou outra qualquer isso não vai acabar registrando uma Data errada?